zilencer/views.py

```diff
@@ -196,7 +196,7 @@ def remote_server_notify_push(
 
     send_apple_push_notification(user_id, apple_devices, apns_payload, remote=True)
 
-    return json_success()
+    return json_success({"android_num": len(android_devices), "apple_num": len(apple_devices)})
```

Comment:
Maybe `total_android_devices: ` and `total_apple_devices` would read more naturally?  

(We like to avoid putting abbreviations like `num` in the API, and also go for variables that read naturally left-to-right)

Reply:
`total_android_devices` and `total_apple_devices` SGTM!